solr/solr-ref-guide/src/index.adoc

```diff
@@ -124,6 +125,8 @@ The *<<getting-started.adoc#getting-started,Getting Started>>* section guides yo
 *<<legacy-scaling-and-distribution.adoc#legacy-scaling-and-distribution,Legacy Scaling and Distribution>>*: This section tells you how to grow a Solr distribution by dividing a large index into sections called shards, which are then distributed across multiple servers, or by replicating a single index across multiple services.
 
 *<<circuit-breakers.adoc#circuit-breakers,Circuit Breakers>>*: This section talks about circuit breakers, a way of allowing a higher stability of Solr nodes and increased service level guarantees of requests that are accepted by Solr.
+
+*<<rate-limiters.adoc#rate-limiters,Request Rate Limiters>>*: This section talks about request rate limiters, a way of guaranteeing throughput per request type and dedicating resource quotas by resource type.
```

Comment:
can we specify that this is per JVM/Solr instance?

Reply:
I had added that in rate-limiters.adoc, added here as well.